Lib/http/server.py

```diff
@@ -132,9 +139,9 @@
 
 DEFAULT_ERROR_CONTENT_TYPE = "text/html;charset=utf-8"
 
-class HTTPServer(socketserver.TCPServer):
 
-    allow_reuse_address = True    # Seems to make sense in testing environment
+class HTTPServer(socketserver.TCPServer):
+    allow_reuse_address = True  # Seems to make sense in testing environment
```

Comment:
> I took the liberty of defining a single style in Lib/http/server.py file, otherwise everything would have been completely confusing.

Please revert these changes. Even if it could be confusing, new code should limit itself to cosmetic changes as it makes the diff harder to inspect, as well as backports harder.

Reply:
i revert correct style commit